src/CommunityToolkit.Maui.MediaElement/Services/MediaControlsService.android.cs

```diff
@@ -160,16 +160,15 @@ void OnReceiveUpdatesPropertyChanged(object? sender, PropertyChangedEventArgs e)
 		}
 		notification.ClearActions();
 		notification.AddAction(actionPrevious);
-		if (receiveUpdates.Action is ACTION_PLAY)
+		if (receiveUpdates.Action is ActionPlay)
 		{
 			notification.AddAction(actionPause);
 		}
-		if (receiveUpdates.Action is ACTION_PAUSE)
+		if (receiveUpdates.Action is ActionPause)
 		{
 			notification.AddAction(actionPlay);
 		}
 		notification.AddAction(actionNext);
-		notification.Build();
```

Comment:
This is an immutable operation. It basically does nothing so I think it's better to remove it.